ControlRoomApplication/ControlRoomApplicationTest/EntityControllersTests/RadioTelescopeControllerTest.cs

```diff
@@ -1185,6 +1185,39 @@ public void TestAllTelescopeMovements_RunABunchOfCommandsAtTheSameTime_OnlyOneCo
             Assert.AreEqual(1, successes);
             Assert.AreEqual(threadCount - 1, alreadyMovings);
         }
+
+        /// <summary>
+        /// Test method that checks for discrepancy between motor and absolute encoders (to see if motor skips steps/is behind)
+        /// </summary>
+        [TestMethod]
+        public void TestCompareMotorAndAbsoluteEncoders()
+        {
+            // If equal
+            Orientation motor = new Orientation(0.0, 0.0);
+            Orientation absolute = new Orientation(0.0, 0.0);
+            Assert.IsTrue(TestRadioTelescopeController.CompareMotorAndAbsoluteEncoders(motor, absolute));
+
+            // If minor discrepancy
+            motor.Elevation = 10.02;
```

Comment:
Find a way to integrate the discrepancy constant into the numbers you set. That way if we ever change the constant, we don't have to update the unit test as well.

- [ ] Integrate discrepancy constant into orientation values